extract-jvm

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+if [ -z $JAVA_HOME ]; then
+    echo -e "Please define JAVA_HOME and try again"
```

Comment:
i don't seem to have a `$JAVA_HOME` with an openjdk-11-headless install, is there a typical debian-flavor path it'd be at?

Reply:
`export JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64`